package.json

```diff
@@ -30,7 +26,7 @@
     "react-table": "7.5.0",
     "react-tooltip": "4.2.10",
     "regenerator-runtime": "^0.13.7",
-    "rx": "^4.1.0"
```

Comment:
`rx` is never used am I right?

Reply:
doesn't seem so

Reply:
Doesn't complain at build time and on storybook so I guess it's fine to remove it 👍